packages/jaeger-ui-components/src/TraceTimelineViewer/SpanBarRow.tsx

```diff
@@ -430,14 +430,15 @@ export class UnthemedSpanBarRow extends React.PureComponent<SpanBarRowProps> {
               nameWrapperMatchingFilter: isMatchingFilter,
             })}
           >
-            <SpanTreeOffset
-              childrenVisible={isChildrenExpanded}
-              span={span}
-              onClick={isParent ? this._childrenToggle : undefined}
-              hoverIndentGuideIds={hoverIndentGuideIds}
-              addHoverIndentGuideId={addHoverIndentGuideId}
-              removeHoverIndentGuideId={removeHoverIndentGuideId}
-            />
+            <div data-testid="span-tree-offset" onClick={isParent ? this._childrenToggle : undefined}>
```

Comment:
don't think we should add this extra wrapping `div`. should probably interact with the elements in the test in a different way.